INSTALL

```diff
@@ -4,6 +4,9 @@ This is the official Kaldi INSTALL. Look also at INSTALL.md for the git mirror i
 (1)
 go to tools/  and follow INSTALL instructions there.
 
-(2) 
+(2)
 go to src/ and follow INSTALL instructions there.
 
+Alternatively, a new cmake based build system is supported with faster build speed.
+Note, it may not be well tested and some features are missing currently.
```

Comment:
Can you be clearer about what the options are?  E.g. is the cmake thing a replacement for (1) and (2) or just (2)?

You could structure it as:
 
Option A:  "make"-based build (this is well tested on multiple platforms)
...

Option B: "cmake"-based build.  This has been added more recently and
   may not be as well tested.
...

Reply:
@danpovey Any suggestion? There a "Resolve conversation". And I am wondering, for github best practice, should you or me click it?

Reply:
@cloudhan, I'd go with the original commenter marking a conversation resolved by default, unless it's a truly trivial case (e. g., if you implemented their suggestion in code as-is). My thinking is, if I have some reservations and start a sub-discussion, then it's upon me to drop these reservation, given the arguments (or come to other solution, but clearly consensual). Again, I'm pretty fine with trivial matters (e.g, I raised doubts but obviously missed that a variable was initialized, and someone pointed at this fact). But for less than trivial question, the discussion stewardship, I think, should be on the topic starter.

Reply:
@kkm000 got it, that make sense. Thanks for commenting.